docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory authentication (Preview)
+description: How to configure Teleport Database Access with Microsoft SQL Server with Active Directory authentication.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Microsoft SQL Server with Active Directory authentication
+  is available starting from Teleport `9.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Database access for Microsoft SQL Server is currently in a Preview mode and
+  does not include audit logging of database query activity.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to your SQL Server using Active Directory authentication.
+- Connect to the database server through Teleport.
+
+This guide will focus on Amazon RDS for SQL Server using AWS-managed Active
+Directory authentication.
+
+## Prerequisites
+
+- Teleport version >= `9.0`.
+- A SQL Server database with Active Directory authentication enabled.
+- A Windows machine joined to the same Active Directory domain as the database.
+- A Linux node joined to the same Active Directory domain as the database. This
```

Comment:
Should we capitalize "node" here? I.e., is this a Teleport Node or a Linux VM more generally?